README.md

```diff
@@ -655,7 +654,6 @@ $date->isExtraWorkday() or ($date->isWeekday() and !$date->isHoliday())
 #### setHolidayGetter
 
 Customize the way to determine if a date is a holiday and which one it is.
- 
```

Comment:
Please keep those blank lines, it makes it easier to read the readme source code.